builtin/credential/github/path_login.go

```diff
@@ -203,14 +211,27 @@ func (b *backend) verifyCredentials(ctx context.Context, req *logical.Request, t
 		orgOpt.Page = resp.NextPage
 	}
 
+	orgLoginName := ""
 	for _, o := range allOrgs {
-		if strings.EqualFold(*o.Login, config.Organization) {
+		if o != nil && *o.ID == config.OrganizationID {
```

Comment:
Ditto comment above about pointer dereference. Might want to use `o.GetID()` to be safe. There may be some other places in the code that I'm missing too.

Reply:
thanks! I just double checked and these are the only two places.